CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# Changelog
+
+## Unreleased
+
+### Breaking Changes
+
+- None
+
+### Added
+
+- None
+
+### Fixed
+
+- None
+
+## 3.0.0 - 2020-06-10
+
+### Breaking Changes
+
+- Cucumber 3 support removed (#762)
```

Comment:
Hi! I believe cucumber 3 should still work?

Reply:
ah yes ... this was just the sorting part

Reply:
https://github.com/grosser/parallel_tests/pull/766